src/Fixer/ClassNotation/ProtectedToPrivateFixer.php

```diff
@@ -82,7 +81,7 @@ public function getPriority()
      */
     public function isCandidate(Tokens $tokens)
     {
-        return $tokens->isAnyTokenKindsFound(Token::getClassyTokenKinds());
+        return $tokens->isTokenKindFound(T_CLASS);
```

Comment:
not handled by the fixer; https://github.com/SpacePossum/PHP-CS-Fixer/blob/ffc59269d3750ffd71db6fbe2d5dad2f87a2516b/src/Fixer/ClassNotation/ProtectedToPrivateFixer.php#L34
Besides I think you can safely change a `protected` method to `private` in a Trait.